tests/unit/matrix/test_assembler.py

```diff
@@ -1,12 +1,14 @@
 # encoding: utf-8
 
-"""Unit test suite for `cr.cube.matrix.assembler` module."""
+"""Unit test suite for `cr.cube.cubepart assembly` module."""
 
+import mock
 import numpy as np
 import pytest
 
 from cr.cube.cube import Cube
-from cr.cube.dimension import Dimension, _Element, _OrderSpec, _Subtotal, _Subtotals
+from cr.cube.cubepart import _Slice
```

Comment:
Should we move these tests that depend on `_Slice` to `tests/unit/test_cubepart.py`?

Reply:
I kept these in test_assembler just for some modularity; although they are now testing methods on the _Slice/_Strand object, they all still call `assemble_*` at some point and it seemed good to keep that separate.